src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1443,16 +1455,14 @@ private IUsingOperation CreateBoundUsingStatementOperation(BoundUsingStatement b
             return new LazyUsingStatement(resources, body, _semanticModel, syntax, type, constantValue, isImplicit);
         }
 
-        private IExpressionStatementOperation CreateBoundThrowStatementOperation(BoundThrowStatement boundThrowStatement)
+        private IThrowOperation CreateBoundThrowStatementOperation(BoundThrowStatement boundThrowStatement)
```

Comment:
Was this decided at the design review meeting? If so, is there a bug tracking the removal of the rest of the uses of IExpressionStatementOperation?

Reply:
> Was this decided at the design review meeting?

This wasn't an explicit decision at the meeting, however this change is in the general spirit of eliminating explicit distinction between expressions and statements.